packages/react/src/components/browserPreview/BrowserPreviewEmpty.tsx

```diff
@@ -15,7 +15,7 @@ export const BrowserPreviewEmpty: React.FunctionComponent<BrowserPreviewEmptyPro
             'cursor-pointer': onClick,
         })}
     >
-        {image ?? <Svg svgName="arrowLeftReturn" className="block" />}
+        {image ?? <Svg svgName="arrowLeftReturn" svgClass="icon mod-info" className="block" />}
```

Comment:
Should we switch to something like <ArrowLeftSize32Px /> ?

Reply:
I don't understand your comment 😛

Reply:
This should be changed to an illustration ideally, let's do that later.